src/odemis/util/test/img_test.py

```diff
@@ -1095,8 +1096,42 @@ def getSubData(dast, zoom, rect):
         
         os.remove(FILENAME)
 
+
+class TestRotateImage(unittest.TestCase):
+
+    def test_rotate_image(self):
+        """
+        Verify that rotating an image around a center of rotation results in
+        the correct metadata being set on the image.
+        """
+
+        # Expected position calculated with the equation:
+        # exp_pos_x = cos(alpha) * (pos_x - cor_x) - sin(alpha) * (pos_y - cor_y) + cor_y
+        # exp_pos_y = sin(alpha) * (pos_x - cor_x) + sin(alpha) * (pos_y - cor_y) + cor_y
+        test_data = [
+            # pos:  {center of rot, rotation,  expected position}
+            {"pos": (0, 0), "cor": (0, 0), "rot": 10, "exp_pos": (0, 0)},
+            {"pos": (0, 1), "cor": (0, 0), "rot": 90, "exp_pos": (-1, 0)},
+            {"pos": (0, 1), "cor": (0, 0), "rot": -90, "exp_pos": (1, 0)},
+            {"pos": (123e-6, 24e-7), "cor": (123.0e-6, 2.4e-6), "rot": 5.7, "exp_pos": (123.0e-6, 2.4e-6)},
+            {"pos": (123e-6, 24e-7), "cor": (10.0e-6, 24e-6), "rot": 5.7, "exp_pos": (1.24586586e-4, 1.37299314e-05)},
+            {"pos": (123e-6, 24e-7), "cor": (10.0e-6, 24e-6), "rot": -5.7, "exp_pos": (1.20295973e-4, -8.7163320e-06)},
+        ]
+
+        data = numpy.random.rand(51, 76)
```

Comment:
The data content doesn't matter, right? So let's just just `numpy.ones(51, 76)`, because "random" make it sound like we will have different behaviour depending on the random seed, which is not set...